playbooks/roles/airship-configure-worker/tasks/main.yml

```diff
@@ -58,9 +56,9 @@
     - skip_ansible_lint
 
 - name: Enroll node for Airship Openstack compute plane
-  shell: |
-    kubectl label nodes {{ inventory_hostname }} openstack-compute-node=enabled
-    kubectl label nodes {{ inventory_hostname }} openvswitch=enabled
+  command: |
```

Comment:
Can you merge the calls here like line 34 to fix break?

Reply:
Good catch!